src/components/molecules/EventModal/EventModal.tsx

```diff
@@ -51,6 +54,14 @@ const EventModal = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [currentEvent]);
 
+  useEffect(() => {
+    if (notes) {
+      if (notes.length === 0) {
+        setInputFieldText('');
+      }
+    }
```

Comment:
At the moment it's not abdunently clear why this is here and what it does. 
Could you please extract it out function/const so that the name can serve as to some explanation?